src/tools/analyzer/modules/request.py

```diff
@@ -188,9 +194,14 @@ def print_formatted_verbose(self, source, patterns):
                 if plugin:
                     print("   - " + plugin)
                 if name:
-                    print("       - " + name[:-2])
+                    # Avoid duplicate CID output with the same CR #
+                    if cr not in self.done:
+                        print("       - " + name[:-2])
+                        self.done = cr
                 if (id and ("UID" in cidline or "GID" in cidline)):
-                    print("       - " + id)
+                    if cr not in self.done:
+                        print("       - " + id)
+                        self.done = cr
```

Comment:
What does `done` store?
Shouldn't it be `+=` and not `=`?

Reply:
Yes you are right, it stores a the CR # which is checked against to suppress duplicates in certain cases. Fixed.